manifests/config.pp

```diff
@@ -9,6 +9,9 @@
   $workers                       = $squid::workers,
   $acls                          = $squid::acls,
   $http_access                   = $squid::http_access,
+  $http_port                     = $squid::http_port,
```

Comment:
For the hash of acl you used a plural parameters *acl* so should be *http_ports* ?